proj-android/PowerAuthLibrary/src/main/java/io/getlime/security/powerauth/sdk/PowerAuthSDK.java

```diff
@@ -1652,10 +1634,12 @@ public void onCancel() {
      * @param title Dialog title.
      * @param description Dialog description.
      * @param callback Callback with the authentication result.
+     * @return {@link ICancelable} object associated with the biometric prompt.
      */
     @RequiresApi(api = Build.VERSION_CODES.M)
-    public void authenticateUsingFingerprint(Context context, FragmentManager fragmentManager, String title, String description, final IFingerprintActionHandler callback) {
-        authenticateUsingFingerprint(context, fragmentManager, title, description, false, callback);
+    @NonNull
+    public ICancelable authenticateUsingFingerprint(Context context, FragmentManager fragmentManager, String title, String description, final IBiometricAuthenticationCallback callback) {
```

Comment:
Shouldn't we rename to `authenticateUsingBiometry`?

Reply:
You're right.